src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/movies';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard movie={movie} />
```

Comment:
? what about using `key` when `.map` in JSX
https://reactjs.org/docs/lists-and-keys.html#embedding-map-in-jsx

Reply:
Okay, fixed. Thank you.